app/views/admin/valuators/search.html.erb

```diff
@@ -29,7 +29,12 @@
                 <%= t("admin.valuators.index.no_description") %>
               <% end %>
             <td>
-              <% unless user.valuator? %>
+              <% if user.valuator? %>
+                <%= link_to t("admin.valuators.valuator.delete"),
```

Comment:
Not sure why this changes appear here, seems like your branch isn't rebased with master, and some commits from angel and xuanxu came along on the PR making it a bit more difficult to review

Reply:
Im working on it